proto/tendermint/abci/types.proto

```diff
@@ -51,6 +51,7 @@ message RequestInfo {
   string version       = 1;
   uint64 block_version = 2;
   uint64 p2p_version   = 3;
+  string abci_version = 4;
```

Comment:
Does this need to be a string? can't we use int64 like we do with `block_version` and `p2p_version`?

Reply:
This field is populated from ABCISemVer, which uses semantic versioning. ABCISemVer is defined this way also in v0.34.x and probably in earlier releases.
So I am for not changing the versioning scheme.
However, I'll take the opp for bumping it now (to signal inclusion of `PrepareProposal` and `ProcessProposal`)